src/chrome/content/rules/Dragonsmoke.cloud.xml

```diff
@@ -0,0 +1,15 @@
+<!--
+	Nonfunctional hosts in *.dragonsmoke.cloud:
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
+-->
+<ruleset name="Dragonsmoke.cloud">
+	<target host="dragonsmoke.cloud" />
+	<target host="www.dragonsmoke.cloud" />
+
+	<rule from="^http://(www\.)?dragonsmoke\.cloud/" to="https://www.dragonsmoke.cloud/" />
```

Comment:
The server redirects you to www no matter what, so there's no reason we should redirect to the non-www version. Might as well just skip straight to www.